testsuite/features/secondary/srv_content_lifecycle.feature

```diff
@@ -162,16 +163,16 @@ Feature: Content lifecycle
   Scenario: Cleanup: remove the created channels for Uyni
```

Comment:
not part of your PR but there is a typo here, can you change it to `Uyuni` please?